setup.py

```diff
@@ -63,10 +62,10 @@ def has_environment_marker_platform_impl_support():
         'Topic :: Software Development :: Libraries :: Application Frameworks',
         'Topic :: Software Development :: Libraries :: Python Modules',
     ],
-    python_requires='>=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*',
+    python_requires='>=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*',
     install_requires=[
-        'Twisted>=13.1.0;python_version!="3.4"',
-        'Twisted>=13.1.0,<=19.2.0;python_version=="3.4"',
+        'Twisted>=13.1.0;python_version=="2.7"',
+        'Twisted>=15.5;python_version>="3.5"',
```

Comment:
Bumped Twisted to `>=15.5` on py35+, according to https://github.com/scrapy/scrapy/blob/master/docs/news.rst#scrapy-110-2016-05-11 (emphasis mine)
> Scrapy 1.1 has beta Python 3 support (**requires Twisted >= 15.5**)